gst/Makefile.am

```diff
@@ -6,8 +6,8 @@ plugin_LTLIBRARIES = libgstdroid.la
 
 libgstdroid_la_CFLAGS = $(GST_CFLAGS) $(NGI_CFLAGS) $(EGL_CFLAGS) \
                         $(NGM_CFLAGS)\
-	                -I$(top_builddir)/gst-libs/ \
-			-I/usr/include/droidmedia/ \
+                        -I$(top_builddir)/gst-libs/ \
+                        -I/usr/include/droidmedia/ \
                         -Idroidcamsrc/ \
```

Comment:
if you would like to fix whitespace, I'd suggest to create a separate "cleanup" commit (without tag and bugref)

Reply:
I'll just leave it for now.

Reply:
It's just tab-space conversion, in vim shows up fine with ts=8, and github borks it due to diff :)